.github/workflows/docker-vulnerability-scan.yml

```
@@ -6,7 +6,7 @@ on:
     - cron: "0 4 * * *"
 
 env:
-  REGISTRY: ${{ secrets.AWS_ACCOUNT }}.dkr.ecr.ca-central-1.amazonaws.com/scan-websites
+  ECR_REGISTRY: ${{ secrets.AWS_ACCOUNT }}.dkr.ecr.ca-central-1.amazonaws.com/scan-websites
```

We can keep this as just the `REGISTRY` env var here
```suggestion
  REGISTRY: ${{ secrets.AWS_ACCOUNT }}.dkr.ecr.ca-central-1.amazonaws.com/scan-websites
```